content/filters/array_filters.md

```diff
@@ -0,0 +1,186 @@
+# Array Filters
+
+Array filters are used to modify the output of arrays.
+
+1. TOC
+{:toc}
+
+## join
+
+Joins the elements of an array with the character passed as the parameter. The result is a single string.
+
+#### Input
+
+~~~ liquid
+  {{ product.tags | join: ', ' }}
+~~~
+
+#### Output
+
+~~~
+tag1, tag2, tag3
+~~~
+
+## first
+
+Returns the first element of an array.
+
+#### Input
+
+~~~ liquid
+  <!-- product.tags = "sale", "mens", "womens", "awesome" -->
```

Comment:
I think it would be good to use BookingSync related words.
`mens`, `womens` can be used for clothes, but in BS context it looks strange